build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
 }
 
 plugins {
-    id 'nebula.netflixoss' version '3.2.3'
+    id 'nebula.netflixoss' version '3.3.0'
```

Comment:
got it , i clone the master branch. when i compile the project , get some error. 
i fix the issue by updating the version.